lib/matplotlib/axes/_axes.py

```diff
@@ -3104,29 +3104,32 @@ def pie(self, x, explode=None, labels=None, colors=None,
             A sequence of colors through which the pie chart will cycle.  If
             *None*, will use the colors in the currently active cycle.
 
+        hatch: str or list, default: None
+            Hatching pattern applied to all pie wedges or list of patterns that
+            are applied to each wedge. For supported patterns, see
+            :ref:`hatch_style_reference`
+
+            .. versionadded:: 3.7
+
         autopct : None or str or callable, default: None
             If not *None*, is a string or function used to label the wedges
             with their numeric value.  The label will be placed inside the
             wedge.  If it is a format string, the label will be ``fmt % pct``.
             If it is a function, it will be called.
 
         pctdistance : float, default: 0.6
-            The ratio between the center of each pie slice and the start of
-            the text generated by *autopct*.  Ignored if *autopct* is *None*.
+            The radial distance at which the the text generated by *autopct* is
+            drawn. To draw the text outside the pie, set *pctdistance*>*radius*
+            (default=1). Ignored if *autopct* is *None*.
+
+        labeldistance : float or None, default: 1.1
+            The radial distance at which the pie labels are drawn. To draw the
+            label inside the pie, set  *labeldistance*<*radius* (default=1).
+            If set to ``None``, labels are only drawn if a legend is created.
```

Comment:
I do not think this is fully correct. Looking at 
```
xt = x + labeldistance * radius * math.cos(thetam)
```
etc, labeldistance is multiplied with radius, so this is a ratio (although I think that the radial distance is a better concept). What about "relative radial distance"? And then skipping the relation to `*radius*`.

`If set to ``None``, labels are only drawn if a legend is created.` gives me the impression that they are drawn as labels if a legend is created, which I guess is not the case (based on the previous doc-string and the fact that drawing the labels is not really needed if a legend is present...).